scheds/rust/scx_p2dq/src/bpf/types.h

```diff
@@ -61,10 +61,12 @@ struct task_p2dq {
 	u32			node_id;
 	u64			used;
 	u64			last_dsq_id;
+	u64			last_cpu_dsq_id;
 	u64 			last_run_started;
 	u64 			last_run_at;
 	u64			llc_runs; /* how many runs on the current LLC */
 	int			last_dsq_index;
+	s32			last_cpu;
```

Comment:
Are you saving last_cpu in the task context for efficiency reasons? Why not using `scx_bpf_task_cpu(p)` directly?